xbmc/GUIInfoManager.cpp

```diff
@@ -7941,32 +7941,18 @@ std::string CGUIInfoManager::GetMultiInfoLabel(const GUIInfo &info, int contextW
   }
   else if (info.m_info == PLAYER_FINISH_TIME)
   {
-    CDateTime time;
-    CEpgInfoTagPtr currentTag(GetEpgInfoTag());
-    if (currentTag)
-      time = currentTag->EndAsLocalTime();
-    else
-    {
-      time = CDateTime::GetCurrentDateTime();
-      int playTimeRemaining = GetPlayTimeRemaining();
-      float speed = g_application.m_pPlayer->GetPlaySpeed();
-      if (speed >= 0.75 && speed <= 1.55)
-        playTimeRemaining /= speed;
-      time += CDateTimeSpan(0, 0, 0, playTimeRemaining);
-    }
+    CDateTime time(CDateTime::GetCurrentDateTime());
+    int playTimeRemaining = GetPlayTimeRemaining();
+    float speed = g_application.m_pPlayer->GetPlaySpeed();
+    if (speed >= 0.75 && speed <= 1.55)
```

Comment:
This test is implemented at 7 locations. Consider moving it into a new property `CApplicationPlayer::GetIsPlaying() const`. This makes it easier to guarantee consistent changes of 0.75 and 1.55 and removes necessity for temporary variables in 3 cases.